docs/plugins/Verifier/cosign.md

```diff
@@ -0,0 +1,385 @@
+---
+sidebar_position: 2
+---
+
+# Cosign
```

Comment:
Suggestion to add a index for this page at the top, as there are many topic covered, having a index will help customer navigate.
- Concepts/Components
    -  Keys for validation (keypair / keyless)
    - Verifiers
    - TrustPolicy
   
- KeyPair validation
    k8s Configuration
    cli config ( coming soon)
 -Keyless validation

Reply:
thanks just added it